.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
To ensure that all instances of `db.sqlite3` are ignored throughout the repository, you should remove the leading slash. Change `db.sqlite3` to `**/db.sqlite3` to ignore it in all directories.